meta-ostro/classes/image-dsk.bbclass

```diff
@@ -26,13 +26,16 @@
 
 # The mask used is dsk_xz instead of dsk.xz because of
 # https://bugzilla.yoctoproject.org/show_bug.cgi?id=9076
-IMAGE_TYPEDEP_dsk.xz = "dsk"
+IMAGE_TYPEDEP_dsk_xz = "dsk"
 IMAGE_TYPES_MASKED += "dsk_xz"
 
 IMAGE_TYPEDEP_dsk.vdi = "dsk"
 IMAGE_TYPES_MASKED += "dsk.vdi"
 
-IMAGE_DSK_ACTIVE = "${@bb.utils.contains_any('IMAGE_FSTYPES', 'dsk dsk_xz dsk.vdi', True, False, d)}"
+IMAGE_TYPEDEP_dsk.vdi_xz = "dsk"
```

Comment:
No, the anomaly is the underscore, see the bug report I mentioned.
I'd rather not let it spread, and instead revert it back to a dot, once that bug is fixed.